components/SummariesHeader.vue

```diff
@@ -0,0 +1,97 @@
+<script setup>
+import { ref, onMounted } from 'vue';
+
+// Initialize reactive references for tracking the count of products, orders, and users.
+const products = ref(0);
+const orders = ref(0);
+const customers = ref(0);
+
+// Asynchronous function to fetch data from API and update state
+const fetchData = async () => {
+
+    try {
+        const responses = await Promise.all([
+            fetch('https://fakestoreapi.com/products'), // Fetch list of products
+            fetch('https://fakestoreapi.com/carts'), // Fetch list of orders
+            fetch('https://fakestoreapi.com/users') // Fetch list of users
+        ]);
+        const data = await Promise.all(responses.map(res => res.json()));
+
+        // Update the state with the count of products, orders, and users
+        products.value = data[0].length;
+        orders.value = data[1].length;
+        customers.value = data[2].length;
+    } catch (error) {
+        console.error("There was an error fetching the data", error);
+    }
+};
+
+onMounted(() => {
+    fetchData();
+});
+</script>
+
+<template>
+    <div class="mt-10 flex justify-center">
+        <div class="stats stats-horizontal max-[340px]:stats-vertical shadow bg-white">
+
+            
+            <div class="stat p-4">
```

Comment:
Create resusable component based on this div with i.e props: title, value, description, color, svg